### DISCUSSION TITLE
MET Stat-Analysis Tool

### DISCUSSION THREAD

**Discussion by mjohnson52** [dtcenter/METplus#1946] [Statistical Computation] [answered]
Is there a way to generate SSVAR output using Stat-Analysis on ensemble data for one variance bin? I am using the tool to aggregate hourly ensemble_stat output into one clean output file (specifically aggregate-stat using ORANK input), but by default the variance bins are partitioned by 0.05. This requires an additional external computation to calculate stats (e.g., RMSE, bias) over the period of interest.

**Comment by CPKalb** [comment, marked as answer]
Hi:

@JohnHalleyGotway checked the logic for Stat-Analysis, and it just uses the bin sizes it finds in the input files to compute aggregated statistics.  Unfortunately, there isn't a mechanism for changing the bin sizes when aggregating the ssvar lines in Stat-Analysis.  This could be something we could think about enhancing in the future, but it hasn't been previously requested.

METviewer does support the ability to change bin sizes when aggregating ssvar data.  Do you have the ability to upload data to METviewer?  If not, you can change the bin sizes output in ssvar lines from Ensemble-Stat using the ens_ssvar_bin_size option in the configuration file, which is described in this [section](https://met.readthedocs.io/en/latest/Users_Guide/ensemble-stat.html#practical-information).

There is also a single RMSE value that comes out of Ensemble-Stat in the ecnt line and could be aggregated.

Christina

**Reply by mjohnson52** [reply]
Christina

Thank you for the helpful answer. I changed the bin size in Ensemble-Stat using ens_ssvar_bin_size and used stat-analysis' aggregate job to calculate RMSE in the manner I wanted. 

I cannot aggregate RMSE in the ecnt which I believe is related to this previous discussion: https://github.com/dtcenter/METplus/discussions/1632.

**Reply by CPKalb** [reply]
Ah, yes.  I know we were working on a fix for the ecnt aggregation issue.  Let me check the status of this work.

Christina

**Reply by CPKalb** [reply]
It sounds like the aggregation error on ecnt lines is updated in the repository, but hasn't been included in the 10.1.3 [bugfix release](https://github.com/dtcenter/MET/milestone/79) yet.  

I'm going to go ahead and lock this discussion since it's marked as answered.  If you have other questions, feel free to start a new discussion.

Christina
